test/run

```diff
@@ -2,7 +2,4 @@
 # This is the expected entry point for Cockpit CI; will be called without
 # arguments but with an appropriate $TEST_OS
 
-# mock compose creation calls to backend for testing purposes
```

Comment:
Please break the overly long line in the commit message.

Reply:
Updated. Thanks @martinpitt.